src/openrct2/core/FileStream.hpp

```diff
@@ -123,25 +130,51 @@ class FileStream final : public IStream
 
     void Seek(sint64 offset, sint32 origin) override
     {
+#if defined(_MSC_VER)
         switch (origin) {
         case STREAM_SEEK_BEGIN:
-            SDL_RWseek(_file, offset, RW_SEEK_SET);
+            _fseeki64(_file, offset, SEEK_SET);
```

Comment:
What about conditionally defining `fseek64` to either `_fseeki64`, `fseeko` or`fseeko64`?